tests/torchtune/models/test_lora_llama2.py

```diff
@@ -171,3 +180,100 @@ def test_lora_llama2_state_dict_parity(
         )
         assert not unexpected
         assert all(["lora" in key for key in missing])
+
+    def test_lora_linear_quantize_base(self):
+        model = self.get_lora_llama2(
+            lora_modules=["q_proj", "v_proj", "k_proj", "output_proj"],
+            apply_lora_to_mlp=True,
+            # quantize_base
+            apply_lora_to_output=False,
+            vocab_size=50,
+            quantize_base=True,
+            embed_dim=512,
```

Comment:
Just curious: do you need a diff embed_dim for some particular reason?

Reply:
Yeah, the defaults for nf4 tensor is that the tensor dim needs to be divisible by the block size, which is 256. So just set 512 here (so the division isn't trivial and we actually end up with multiple blocks to quantize).

In the future, we can probably have some way where we override the NF4Tensor defaults / configure them to something more sensible for torchtune's workloads.